apps/cowswap-frontend/src/modules/twap/updaters/index.tsx

```diff
@@ -15,12 +15,12 @@ import { TwapOrdersUpdater } from './TwapOrdersUpdater'
 import { useTwapSlippage } from '../hooks/useTwapSlippage'
 
 export function TwapUpdaters() {
-  const { chainId, account } = useWalletInfo()
+  const { account } = useWalletInfo()
   const isSafeApp = useIsSafeApp()
-  const composableCowContract = useComposableCowContract()
+  const { contract: composableCowContract, chainId: composableCowChainId } = useComposableCowContract()
   const twapOrderSlippage = useTwapSlippage()
 
-  const shouldLoadTwapOrders = !!(isSafeApp && chainId && account && composableCowContract)
+  const shouldLoadTwapOrders = !!(isSafeApp && account && composableCowContract)
```

Comment:
Shouldn't it be replaced by `composableCowChainId`?

```suggestion
  const shouldLoadTwapOrders = !!(isSafeApp && composableCowChainId && account && composableCowContract)
```

Reply:
`composableCowChainId` is truthy, also if it wasn't, the contract would be null